### DISCUSSION TITLE
How can you handle pagination in a graphql query?

### DISCUSSION THREAD

**Discussion by pkaeding** [reposaur/reposaur#31] [Q&A] [answered]
I want to write a policy regarding the members in my github org, which has more than 100 members. I have a graphql query that looks like this (based on [the docs](https://docs.reposaur.com/policy/builtins#githubgraphql)):

```
members_query := `query($org: String!, $after: String!){
  organization(login: $org) {
    memberStatuses(first: 100, after: $after) {
      totalCount
      edges {
        cursor
        nodes {
          user {
            login
          }
        }
      }
    }
  }
}`

members_struct := github.graphql(members_query, {
    "org": input.login
})
```

But, I don't see how to get the next page, and then be able to use the whole list in a policy. Can you give an example of how this might work?

Thanks

**Comment by crqrdotcom** [comment, marked as answer]
Hey @pkaeding 👋🏼 

Unfortunately pagination isn't supported as of now. I've opened #32 for us to implement this, should be straightforward. I'll work on this this week 😉
